lib/shared/get-cache-filename.js

```diff
@@ -1,9 +1,53 @@
+const fs = require('fs');
 const path = require('path');
 
-module.exports = function getCacheFilename(modelConfig) {
+function getCacheFilename(modelConfig, newFile = false) {
     let { slug, cache } = modelConfig;
+    let cacheFiles = [];
+    let cacheFile;
+
+    if (fs.existsSync(cache) && !newFile) {
```

Comment:
I suppose no needs to check `fs.existsSync(cache)` here (getCacheFiles will check it anyway)